rkt/fetch.go

```diff
@@ -54,6 +57,7 @@ func init() {
 	cmdFetch.Flags().BoolVar(&flagStoreOnly, "store-only", false, "use only available images in the store (do not discover or download from remote URLs)")
 	cmdFetch.Flags().BoolVar(&flagNoStore, "no-store", false, "fetch images ignoring the local store")
```

Comment:
I think we should mark `store-only` and `no-store` explicitly as deprecated here and elsewhere.